From fc091e6c42b452109b62f36bd1c5d11a78c3e332 Mon Sep 17 00:00:00 2001
From: xtex <xtexchooser@duck.com>
Date: Sat, 6 Dec 2025 19:55:48 +0800
Subject: [PATCH 083/133] std.Target: add loongarch64 model back for
 compatibility
X-Developer-Signature: v=1; a=openpgp-sha256; l=517; i=xtex@astrafall.org;
 h=from:subject; bh=fnxD4R67OakR1NC6BZloMtV+t+wutcyQardhsG6Ws5A=;
 b=owGbwMvMwCW2U4Ij7wZL9ETG02pJDJn78tivOq01cJQqWbB1Q7vR+t1RZ83UfErKuv0/xv1ax
 f1mjmB+x0QWBjEuBksxRZYiwwZvVp10ftFl5bIwc1iZQIZIizQwMDAwsDDw5SbmlRrpGOmZahvq
 GRrqGOkYMXBxCsBUf+1lZHh5tEteNYc9blKm9/x14pNFErTXz45qW5f140HKkrVHDA0ZGS72H6t
 esOSt8ylB5W2r9SdGPii6fv3oiy2XepfPqEphXMkDAA==
X-Developer-Key: i=xtex@astrafall.org; a=openpgp;
 fpr=7231804B052C670F15A6771DB918086ED8045B91

Change-Id: Ifa74455da3520ec5cfad12f5eeeff95c74d06dfa
---
 lib/std/Target/loongarch.zig | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/lib/std/Target/loongarch.zig b/lib/std/Target/loongarch.zig
index cc49c1cd0063..017f756b9a07 100644
--- a/lib/std/Target/loongarch.zig
+++ b/lib/std/Target/loongarch.zig
@@ -219,4 +219,7 @@ pub const cpu = struct {
             .ual,
         }),
     };
+
+    /// Deprecated; for compatibility only.
+    pub const loongarch64 = la64v1_0;
 };
-- 
2.52.0

